packages/api-catalog-service/.env.example

```diff
@@ -0,0 +1,23 @@
+## Uncomment the lines as needed
+
+NAME=local
+NODE_ENV=local
+PRODUCTION=false
+
+## Database
+DB_PATH=mongo
+DB_NAME=api-catalog
+# DB_USER=
```

Comment:
Should uncomment(or remove) the variables(like: GATEWAY_AUTH_TOKEN). If we don't need them in the local set up then put a comment about the same.

Reply:
updated